trunk/source/RCFGBuilder/src/de/uni_freiburg/informatik/ultimate/plugins/generator/rcfgbuilder/cfg/BoogieIcfgContainer.java

```diff
@@ -76,7 +77,9 @@ public class BoogieIcfgContainer extends ModernAnnotations implements IIcfg<Boog
 	private final Map<String, BoogieIcfgLocation> mEntryNodes;
 	private final Map<String, BoogieIcfgLocation> mExitNode;
 	private final Set<BoogieIcfgLocation> mLoopLocations;
+	private final Set<BoogieIcfgLocation> mLocationsOfInterest;
 	private final Map<String, Set<BoogieIcfgLocation>> mErrorNodes;
+	private final NestedMap2<String, String, BoogieIcfgLocation> mLabelNodes;

```

Comment:
This field is not used anymore and should be removed.

Reply:
https://github.com/ultimate-pa/ultimate/pull/713/commits/201de1901f893ce253a86c04ce85f95870c2a0b0